docs/src/pages/guides/FHIRBulkOperations.md

```diff
@@ -127,16 +132,21 @@ The response returned is 202 if the job is queued or not yet complete.
 Note: If the job is stopped, e.g, due to shutdown of the IBM FHIR Server where the job is running the bulkdata job, then upon polling the status of the bulkdata job, the server restarts the JavaBatch job, and the response to the client is Accepted: 202.
```

Comment:
This may happen on a different IBM FHIR Server node...do you think that's worth mentioning or too much info at this point?

Reply:
Too much info.